docs/pages/_data/services/hdfs.yml

```diff
@@ -7,7 +7,7 @@ techName: HDFS
 # Values specific to certain templates:
 
 install:
-  minNodeCount: five
+  minNodeCount: three
```

Comment:
From slack, sounds like it's indeed five?

Reply:
Does it not depend on the size of the nodes? I'm happy to make it 5 as long as the pre-install notes are then updated accordingly.